qiskit_ibm_runtime/api/auth.py

```diff
@@ -68,24 +68,3 @@ def get_headers(self) -> Dict:
                 f"Unable to retrieve IBM Cloud access token. API Key will be used instead. {ex}"
             )
             return {"Service-CRN": self.crn, "Authorization": f"apikey {self.api_key}"}
-
-
-class QuantumAuth(AuthBase):
```

Comment:
Same comment as with QuantumAccount, when a class is removed we always need to check if it was part of the public interface (luckily it wasn't).